src/Core/Variables/VariableExtractor.php

```diff
@@ -18,6 +18,7 @@ class VariableExtractor
     const ACCESSOR_ARRAY = 'array';
     const ACCESSOR_GETTER = 'getter';
     const ACCESSOR_ASSERTER = 'asserter';
+    const ACCESSOR_ASSERTER_HAS = 'asserterHas';
```

Comment:
Rather than add a new constant I would simply use the existing `ACCESSOR_ASSERTER` and when this is specified, check which of the methods to use. But when defining the accessor you should specify "asserter" if the property has either `is` or `has` prefix.